generator/test_case_generator.go

```diff
@@ -137,6 +137,9 @@ var testTypeToTestConfig = map[string][]testConfig{
 		{testDir: "../../test/stress/windows/logs"},
 		{testDir: "../../test/stress/windows/system"},
 	},
+	"ec2_trace_integration": {
```

Comment:
Could the test just be added to the ec2 linux block? Is there anything specific to the trace matrix that is preventing us from doing that?